shared/fs/common/security-prefs-prompting-hoc.js

```diff
@@ -44,11 +49,12 @@ const displayOnce = ({shouldPromptSecurityPrefs, showSecurityPrefsOnce}) => {
 }
 
 const DesktopSecurityPrefsPromptingHoc = compose(
-  connect(
+  connect<OwnProps, _, _, _, _>(
     mapStateToProps,
     mapDispatchToProps,
     (s, d, o) => ({...o, ...s, ...d})
   ),
+  // $FlowIssue
```

Comment:
@chrisnojima here I'm getting an issue...any ideas? I've tried every permutation of putting those in and/or using `RouteProps` from `util/container`:
```
yarn run v1.9.4
$ /Users/jzila/repos/go/src/github.com/keybase/client/shared/node_modules/.bin/flow
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ fs/common/security-prefs-prompting-hoc.js:57:3

Cannot call compose with compose intermediate value bound to a because:
 • property routePath is missing in object literal [1] in type argument Props [2].
 • property routeProps is missing in object literal [1] in type argument Props [2].

     fs/common/security-prefs-prompting-hoc.js
      54│     mapDispatchToProps,
 [1]  55│     (s, d, o) => ({...o, ...s, ...d})
      56│   ),
      57│   branch(displayOnce, renderNothing)
      58│ )
      59│
      60│ const SecurityPrefsPromptingHoc = isMobile ? (i: any) => i : DesktopSecurityPrefsPromptingHoc

     /private/tmp/flow/flowlib_16cd6bff/react.js
 [2] 152│ declare type React$ComponentType<Props> =
```

Reply:
i don't get this warning w/ 0.86, you still do?

Reply:
Ah, nope. I get a different one with that version.